spog/ui/src/pages/scanner/mod.rs

```diff
@@ -106,7 +131,7 @@ pub fn parse(data: &[u8]) -> Result<SBOM, anyhow::Error> {
             match supported_packages {
                 Some(true) => {}
                 _ => bail!(
-                    "Unsupported packages detected. Supported packages: 'maven', 'gradle', 'npm', 'gomodules', 'pip'"
+                    "The SBOM contains package type(s) not supported by Dependency Analytics. The Dependency Analytics report may be unavailable for this SBOM."
```

Comment:
My immediate question as a user would be: Which are the supported ones?

Reply:
@ctron you're right but I thought you, Dejan, Phil, and Michelle agreed on that "message wording". At least that was my impression from the comment made by Phil here https://issues.redhat.com/browse/TC-1001 xD

Reply:
I am not sure we agreed in any message. But I see your point, the PR fulfills the JIRA. So let's move on until we hear otherwise.